src/main/java/cart/controller/dto/request/product/ProductInsertRequest.java

```diff
@@ -0,0 +1,45 @@
+package cart.controller.dto.request.product;
+
+import cart.domain.Product;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import org.hibernate.validator.constraints.URL;
+
+public class ProductInsertRequest {
+
+    @NotBlank(message = "이름을 다시 입력해주세요.")
+    private final String name;
+
+    @URL(message = "이미지 주소를 다시 입력해주세요.")
+    private final String image;
+
+    @NotNull(message = "가격은 최소 100원, 최대 10000000원입니다.")
```

Comment:
사용자에게 보여줄 메시지라면 예쁘게 보여주면 좋겠죠 😄 
<img width="443" alt="image" src="https://user-images.githubusercontent.com/59511689/235637798-2b06456a-6e20-44a9-bc01-fc6ff7987bf7.png">

```suggestion
    @NotNull(message = "가격은 최소 100원, 최대 10,000,000원입니다.")
```